README.md

```diff
@@ -4,9 +4,9 @@ The Inclusive Coding Environment is a new educational coding tool designed for l
 
 ![inclusive coding environment UI](./inclusive-coding-env.png)
 
-To try the latest development version of the Inclusive Coding Environment, visit our [development build](https://c2lc-dev.web.app/).
+To try the latest development version of the Inclusive Coding Environment, visit our [development build](https://develop-0-6--c2lc-build.netlify.app).
 
-Currently, the Inclusive Coding Environment supports the [Dash](https://www.makewonder.com/robots/dash/) robot. We'll be adding support for new robots in upcoming releases.
+Currently, the Inclusive Coding Environment doesn't support physical robot connection. We'll be adding support for physical robot connections in upcoming releases.
```

Comment:
I'm thinking that we just remove this paragraph.

I think we should also remove this sentence from the opening paragraph: "Learners can create code sequences that control the movement of a Bluetooth-connected robot."